binutils/plan.sh

```diff
@@ -16,7 +17,7 @@ do_prepare() {
 
   # Add explicit linker instructions as the binutils we are using may have its
   # own dynamic linker defaults.
-  dynamic_linker="$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2"
+  dynamic_linker="$(pkg_path_for core/glibc)/lib/ld-linux-x86-64.so.2"
```

Comment:
Personally I prefer the more verbose form.
In theory, somebody could compile the entire stack with clang and then have `clang/glibc` as a dependency.